libyul/backends/wasm/EVMToEWasmTranslator.cpp

```diff
@@ -408,17 +866,23 @@ function gasprice() -> z1, z2, z3, z4 {
 	eth.getTxGasPrice(0)
 	z1, z2, z3, z4 := mload_internal(0)
 }
+function extcodesize_internal(x1, x2, x3, x4) -> r {
+	mstore_internal(0, x1, x2, x3, x4)
```

Comment:
Should use address conversion - or something like `mstore_address()`?

Reply:
done